pkg/validator/validator_test.go

```diff
@@ -5,33 +5,32 @@ import (
 )
 
 var testData = []struct {
-	name string
-	testInput []byte
+	name           string
```

Comment:
Does fmt align the variables like this or was this manual?

Reply:
fmt does it

Reply:
That's pretty gross. Is there a way to disable that specifically?

Reply:
I think it looks worse on the MR diff. Here's what the changes look like on the file itself:

![image](https://user-images.githubusercontent.com/118750525/207671391-d76e4dc5-134a-4707-b48a-6f10e2a8e68a.png)

I don't like so much indentation but that's the go standard and I'm hesitant to deviate from it.

Reply:
Oh okay that's my bad, that looks completely fine.